docs/getting_started.md

```diff
@@ -1,199 +1,102 @@
-```
-
-Only the `node` subsystem is on by default; be aware that the other subsystems track a lot of information and may significantly increase the amount of output that Shadow produces.
-
-The tgen application also logs generally useful statistics, such as file download size and timing information. This information can be parsed from the corresponding log files in the virtual process data directories.
-
-## Parsing and plotting results
-
-Shadow includes some python scripts that can parse some important statistics from the Shadow and TGen messages, including network throughput over time, client download statistics, and client load statistics, and then visualize the results. The following will parse and plot the output produced from the above experiment:
-
-```bash
-# start in the base shadow/ directory
-cd ../..
-# parse the shadow output file
-python3 src/tools/parse-shadow.py --help
-python3 src/tools/parse-shadow.py --prefix results resource/examples/shadow.log
-# plot the results!
-python3 src/tools/plot-shadow.py --help
-python3 src/tools/plot-shadow.py --data results "example-plots"
-```
-
-The `parse-*.py` scripts generate `stats.*.json.xz` files. The (heavily trimmed) contents of `stats.shadow.json` look a little like this.
-
-        nodes:
-            4uthority:
-                recv:
-                    bytes_control_header:
-                        0: 0
-                        1: 0
-                        2: 0
-                    bytes_control_header_retrans:
-                    bytes_data_header:
-                    bytes_data_header_retrans:
-                    bytes_data_payload:
-                    bytes_data_payload_retrans:
-                    bytes_total:
-                send:
-            relay1:
-            relay2:
-        ticks:
-            0:
-                maxrss_gib: 0.010578
-                time_seconds: 5.003849
-            1:
-            2:
-
-The `plot-*.py` scripts generate graphs. Open the PDF file that was created to see the graphed results.
-
-You can also parse and plot the TGen output using the `tgentools` program from the TGen repo. [This page describes how to get started](https://github.com/shadow/tgen/blob/main/doc/Tools-Setup.md).
-
-## Example experiment
-
-Consider a set of experiments where we would like to analyze the effect of changing our nodes' traffic queueing disciplines. We run the following 2 experiments:
-
-```bash
-cd resource/examples/
-rm -rf shadow.data shadow.log qdisc-fifo.data qdisc-fifo.log qdisc-rr.data qdisc-rr.log
-shadow --interface-qdisc fifo --data-directory qdisc-fifo.data shadow.config.xml > qdisc-fifo.log
-shadow --interface-qdisc rr   --data-directory qdisc-rr.data   shadow.config.xml > qdisc-rr.log
-```
+### Simulation Output
 
-To parse these log files, we use the following scripts:
+Shadow will write simulation output to the `shadow.data` directory. Each host has its own directory under `shadow.data/hosts`. For example:
```

Comment:
Something like:
to the data directory (in this example we'll assume the default value of `shadow.data`)
?

Reply:
Added.